control/tests/timeresp_test.py

```diff
@@ -204,6 +204,14 @@ def test_impulse_response(self):
         np.testing.assert_array_almost_equal(
             yy, np.vstack((youttrue, np.zeros_like(youttrue))), decimal=4)
 
+        # discrete time
+        self.siso_tf1
```

Comment:
```suggestion
        sys = self.siso_tf1
```

Should go into separate test function. It is independent of the previous tests.

Reply:
Thanks for catching that test bug. fixed, and moved unit test to a new function